### PR TITLE
Main module

### PR DESCRIPTION
**Pull request by jakearchibald**

### CODE REVIEW

File: src/_includes/defs.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :root {
   --link-color: var(--blue);
 
-  --side-margin: 0.47rem;
+  --side-margin: 0.87rem;
```
Comment (jakearchibald) [inline src/_includes/defs.css]: It seemed a bit tight vs the design

File: src/_includes/layouts/main/style.css [hunk 2]
```diff
@@ -52,9 +52,17 @@ html {
   /* https://github.com/necolas/normalize.css/issues/665 */
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
     Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  font-size: 21px;
   line-height: 2em;
   font-weight: 300;
+  font-size: 14px;
```
Comment (jakearchibald) [inline src/_includes/layouts/main/style.css]: Better for 240px?

File: src/_includes/main-intro/style.css [hunk 3]
```diff
@@ -1,5 +1,181 @@
-.title {
-  font-size: 50px;
+.root {
+  overflow: hidden;
+  position: relative;
+  line-height: normal;
+  background: var(--offwhite);
+  padding: 4.4rem 0;
```
Comment (surma) [inline src/_includes/main-intro/style.css]: Should this be

```suggestion
  padding: var(--vertical-margin) 0;
```

?

File: src/_includes/main-intro/index.njk [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+        <span class="{% className mainIntroCss, 'month' %}">{% confDate conf.start, 'MMMM' %}</span>
+        <a href="#" class="{% className mainIntroCss, 'date-link' %}">
+          {% confDate conf.start, 'DD' %}-{% confDate conf.end, 'DD' %}
+        </a>
+      </span>
+      <a href="#" class="{% className mainIntroCss, 'location-main' %}">Yerba Buena Center for the Arts</a>
+      <span class="{% className mainIntroCss, 'location-sub' %}">San Francisco, CA</span>
+    </p>
+    <div class="{% className mainIntroCss, 'actions' %}">
+      <a href="#" class="{% className mainIntroCss, 'register' %}">Register</a>
```
Comment (surma) [inline src/_includes/main-intro/index.njk]: Technically, this is the “Get Updates” button in the design (and it’s grey). Intentional?

File: src/_includes/main-intro/style.css [hunk 5]
```diff
@@ -1,5 +1,181 @@
...
+}
+
+.cal {
+  padding: 0.9em 0;
+  display: block;
+  position: relative;
+  white-space: nowrap;
+
+  @media (min-width: 720px) {
+    --icon-size: 30px;
```
Comment (surma) [inline src/_includes/main-intro/style.css]: What about this instead?

```suggestion
    --icon-size: 1.5rem;
```

File: src/_includes/main-intro/style.css [hunk 6]
```diff
@@ -1,5 +1,181 @@
...
+
+  @media (min-width: 880px) {
+    transform: none;
+  }
+}
+
+.sign {
+  position: absolute;
+  bottom: 0;
+  right: -30px;
```
Comment (surma) [inline src/_includes/main-intro/style.css]: If we change this to

```suggestion
  left: calc(50% + 36vh);
```

than windows with small heights won’t have an abandoned sign at the right border.

File: src/_includes/main-intro/index.njk (hunk 4 above)
Comment (jakearchibald) [reply src/_includes/main-intro/index.njk]: This is kinda what I mentioned to Adam in chat. I think this is the point we want to get the user to show an interest in attending, or watching live.

Tbh, we should probably change this to "register to attend", or "Get tickets", or something that make the purpose clearer.

Filed https://github.com/GoogleChrome/devsummit/issues/131

File: src/_includes/main-intro/style.css (hunk 3 above)
Comment (jakearchibald) [reply src/_includes/main-intro/style.css]: Not in this case. The padding at this top of the main module is much larger than other modules.

File: src/_includes/main-intro/style.css (hunk 6 above)
Comment (jakearchibald) [reply src/_includes/main-intro/style.css]: Yeah this works much better

**Comment by jakearchibald** [comment]
@surma changes made

**Comment by jakearchibald** [comment]
@surma I'm not hitting the button until you give an approving review

**Comment by jakearchibald** [comment]
THEM'S THE RULEZ

**Pull request merged**